safe-global/safe-wallet-web
# fix: Close transaction modals as soon as the transaction is submitted

## What it solves

Resolves #3675 

## How this PR fixes it

- Uses the unchecked signer when executing transactions

## How to test it

This needs to be tested with MetaMask and at least one other wallet thats not affected e.g. Rabby.
We should also make sure it still works when executing transactions with hardware wallets and nested safes.

The following transaction types need to be tested:
- Spending limit execution
- Bulk execution
- Tx speedup
- Recovery setup
- Recovery proposal

The transaction modal should close when the transaction is executed from the wallet.

## Checklist
